packages/vue/src/components/reset-password.vue

```diff
@@ -16,7 +16,11 @@ const useAuthShared = createSharedComposable(useAuthenticator);
 const { state, send, submitForm } = useAuthShared();
 const { error, isPending } = toRefs(useAuthShared());
 
+const formOverrides = state.context?.config?.formFields?.resetPassword;
+const userOR = formOverrides?.['username'];
```

Comment:
nit: what do you think of `userOverrides` for a bit more readability? I wasn't sure what `OR` mean on first read.
```suggestion
const userOverrides = formOverrides?.['username'];
```

Reply:
I can rename that. #9604434